package-lock.json

```diff
@@ -594,7 +594,6 @@
             "resolved": "https://registry.npmjs.org/@eslint-community/eslint-utils/-/eslint-utils-4.4.1.tgz",
             "integrity": "sha512-s3O3waFUrMV8P/XaF/+ZTp1X9XBZW1a4B97ZnjQF2KYWaFD2A8KyFBsrsfSjEmjn3RGWAIuvlneuZm3CUK3jbA==",
             "dev": true,
-            "license": "MIT",
```

Comment:
Random removals here seem to be a bug fixed in recent versions of npm: https://github.com/npm/cli/commit/effe9109d6bc7828bf916c4dee49b2a53c72f39d

Since we don't really care about the contents of the lockfile as long as it works, I think we can just keep merging these PRs